assignments/prototypes.js

```diff
@@ -102,9 +131,39 @@
   console.log(archer.greet()); // Lilith offers a greeting in Elvish.
   console.log(mage.takeDamage()); // Bruce took damage.
   console.log(swordsman.destroy()); // Sir Mustachio was removed from the game.
-*/
+
 
   // Stretch task: 
   // * Create Villain and Hero constructor functions that inherit from the Humanoid constructor function.  
+function Villain(villainAttributes) {
+  this.name = villainAttributes.name;
+  this.weapon = villainAttributes.weapon;
+  this.healthPoints = villainAttributes.healthPoints;
+  CharacterStats.call(this, villainAttributes);
+}
+
+Villain.prototype = Object.create(Humanoid.prototype);
+
+Villain.prototype.removeHealth = function () { 
```

Comment:
There are a few concepts that are being misused here. First, is the if statement. if statements are going to find the boolean value of whatever is placed within the `()`  But, your use of commas are throwing it an error. I am not sure what's going on with the `[hp]`, but it looks like you were on your way to assigning one variable to the value of another. Unlike objects and arrays, variable copies of stings and numbers will just create a new instance of whatever value. 
``` 
let original = 8;
let copy = original;
copy -= 3;
console.log(copy); //5
console.log(original); //8
``` 

this code block should demonstrate that concept. So if your intention was to remove 2 this.healthPoints, you would either need to assign `hp` back to `this.healthPoints`, or just use `this.healthPoints` throughout.